.eslintrc.js

```diff
@@ -12,12 +12,14 @@ module.exports = {
   rules: {
     ...base.rules,
     // this seems very broken atm, false positives
-    '@typescript-eslint/unbound-method': 'off'
+    '@typescript-eslint/unbound-method': 'off',
+    'sort-keys': 'off',
```

Comment:
There's always never enough fields to have troubles finding fields in an object, but forcing alphabetical order has 2 drawbacks:
1. it prevents semantical order/grouping, which is often more important than alphabetical (which is a kind of random order if you think about it)
2. in case of a field rename, the field might have to be moved up or down the object, which would be considered by git diff as a deletion and addition instead of a modification